_i18n/es.yml

```diff
@@ -1140,24 +1140,24 @@ index:
   why:
     heading: |-
       ## Login.gov es para ti
-    developers: |-
-      #### Desarrolladores de agencias
-
-      Recursos para desarrolladores, soporte en tiempo real y herramientas modernas para ayudarlo a implementar e implementar su aplicación con login.gov
-
-      [Ver guía para desarrolladores](https://developers.login.gov){:class="why-more-info" :target="_blank"}
     individuals: |-
-      #### Individuos
+      ### Individuos
```

Comment:
Not for here, but in seeing these changes, I noticed that because these non-English locale texts still use markdown headings (vs. `<h3 class=` in English), there's no associated graphic with the headings in Spanish and French. 

Between opportunity for desync and the number of changes necessary here to make a markup change to the content, it leads me to sense that it'd be easier to maintain the markup in the templates, then only localize the text fragments (heading, content, link text).

Reply:
Big +1 to this --- I wish we didn't have as much markup in these i18n files as we do

Reply:
Huge +1 to this. Part of my work this sprint is to explore content management options that would move content into easy to manage files and move markup into html templates...and out of yaml files. Options range from markdown files to a federalist-friendly solution like Netlify. The tricky part is creating an easy workflow for translation. Keep an eye out for a future PR.